server-core/src/main/kotlin/com/lightningkite/lightningserver/db/ModelRestUpdatesWebsocket.kt

```diff
@@ -131,13 +132,15 @@ class ModelRestUpdatesWebsocket<USER: HasId<*>?, T : HasId<ID>, ID : Comparable<
             }.filter { it.old != null || it.new != null }
 
             jobs.add(launch {
-                if (toSend.size > 20) {
+                val updates = CollectionUpdates(
+                    updates = toSend.mapNotNull { it.new }.toSet(),
+                    remove = toSend.mapNotNull { it.old.takeIf { _ -> it.new == null }?._id }.toSet()
+                )
+                val size = Serialization.json.encodeToString(CollectionUpdates.serializer(info.serialization.serializer, info.serialization.idSerializer), updates).length
+                if (size >= 24000) {
```

Comment:
This seems rather large if the last one was 20. Is this correct?

Reply:
It's now measuring payload size, not item count